### ISSUE TITLE
avcodec.Rational struct returns contains unexported attributes

### ISSUE THREAD

**Issue by russ666** [targodan/ffgopeg#11]
C-lang struct C.struct_AVRational, presented as avcodec.Rational contains lowercased `num` and `den` attributes, which couldn't be accessed from external package. The only way to access it is by using reflections.

**Comment by targodan** [comment]
Yes, I see. I guess one would need to add exported accessing methods to the struct.

If anyone can to it, feel free to do so, as it may take some time before I can get around to do this.
